docker/back/api/nginx/vhost.conf

```
@@ -0,0 +1,25 @@
+server_tokens off;
+
+log_format docker '$remote_addr - [$time_local] "$request" $status $body_bytes_sent "$http_referer" "$http_user_agent" $request_time "$http_x_forwarded_for"';
+
+server {
+    root /var/www/html/public;
+
+    access_log /var/log/nginx/access.log docker;
+    error_log /var/log/nginx/error.log error;
+
```

```suggestion
add_header X-Request-Id $request_uid;
```
Retourne l'identifiant de requête dans les headers

---

done